### PR TITLE
GUI/tests

### PR DESCRIPTION
**Pull request by GuyomT**
add unit tests for unpack entities

### CODE REVIEW

File: src/zappy_server_src/net_utils/select.c.tmp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+
+
+
```
Comment (LucasTesnier) [inline src/zappy_server_src/net_utils/select.c.tmp:4]: ?????????? What's this file ?

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Del the .tmp file

File: src/zappy_server_src/net_utils/select.c.tmp (hunk 1 above)
Comment (GuyomT) [reply src/zappy_server_src/net_utils/select.c.tmp]: sry, here is the utility of PRs...

**Review by LucasTesnier — APPROVED** [review]
NTR!

**Review by osvegn — APPROVED** [review]
LGTM

**Pull request merged**